### DISCUSSION TITLE
TrackingMore Integration (For Order Tracking)

### DISCUSSION THREAD

**Discussion by HarrySmith01** [inspireui/support#5080] [New Feature/Ideas Requests]
TrackingMore is a tracking service like AfterShip but it is providing best services on lowest price.

Here, you do not need to pay monthly charges, just buy it for once for x number of package and use it untill you do not exceed your credit.

This is a good service for startups and growing business.

https://www.trackingmore.com/api-index.html#authentication

Of if we can integrate below plugin then that too will work:

https://codecanyon.net/item/woocommerce-trackingmore/24008326

**Comment by kodzodeku** [comment]
Sounds cool. This could be very helpful for my business in Africa

**Comment by pulstar** [comment]
I was using Aftership as my tracking service, but now I am using the WooCommerce Orders Tracking plugin (https://codecanyon.net/item/woocommerce-orders-tracking/26062993), which uses TrackingMore by default (but can also use Aftership and some other tracking services). TrackingMore gives more queries and is cheaper than Aftership (see the links below). It also offers the option to track packages from AliExpress, which is important for dropshipping.

https://www.aftership.com/pricing

https://www.trackingmore.com/pricing.html

**Comment by osman-kocak** [comment]
This plugin is better than all
Please check it 
[plugin](https://codecanyon.net/item/woocommerce-shipping-tracking/11363158)

**Reply by pulstar** [reply]
Does this plugin work with the ALD Dropshipping plugin to automatically update the tracking code for packages?

**Comment by chungxon** [comment]
Has anyone used the plugin? https://wordpress.org/plugins/17track/

**Comment by pulstar** [comment]
Is it possible that FluxStore allows us to use other tracking methods besides AfterShip? The application could perhaps open a webview with the URL of the package tracking page on my Wordpress website, passing the order id, user email and tracking number. The ideal is to have the tracking page on my own website and not on an external URL like AfterShip does, so I prefer to use the TrackingMore service or the WooCommerce Shipping Tracking plugin mentioned above.
